src/main/java/com/assignment1/exceptions/RuntimeExceptionCustom.java

```diff
@@ -0,0 +1,11 @@
+package com.assignment1.exceptions;
+
+/** Class for custom exception. */
+public class RuntimeExceptionCustom extends Exception {
```

Comment:
```suggestion
public class GenericApplicationException extends Exception {
```

Reply:
its better to create exceptions with purpose that these basic ones when the application can handle the exceptions not just catching them and logging. (just wanted you to know)